        <html><head></head><body>
        <h1>Report for imdbpy on toolchain
            <a href="index.html">android</a>
        </h1>
        <p>Return code: 5</p>
        <p>Time to run: 00:00:58</p>
        <p>Log output:</p>
        <code><pre><docker run --rm -v /tuscan_data --volumes-from tuscan_data -v /sources --volumes-from sources -v /var/cache/pacman/pkg --volumes-from pkg_cache_android -v logs:/logs make_package_container --sources-directory /sources --shared-directory /tuscan_data imdbpy
:: Synchronizing package databases...
downloading repo.db...
downloading core.db...
downloading extra.db...
downloading community.db...
Found 2002 packages in cache
===> sudo -u tuscan makepkg --nobuild --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: imdbpy 5.0-1 (Fri Nov 13 23:53:36 UTC 2015)
==> Checking runtime dependencies...
==> Installing missing dependencies...
resolving dependencies...
looking for conflicting packages...

Packages (2) sqlite-3.9.2-1  python2-2.7.10-2

Total Installed Size:  75.75 MiB

:: Proceed with installation? [Y/n] 
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
checking available disk space...
installing sqlite...
installing python2...
Optional dependencies for python2
    tk: for IDLE
    python2-setuptools
    python2-pip
==> Checking buildtime dependencies...
==> Installing missing dependencies...
:: There are 2 providers available for python2-distribute:
:: Repository repo
   1) python2-setuptools
:: Repository extra
   2) python2-setuptools

Enter a number (default=1): 
resolving dependencies...
looking for conflicting packages...

Packages (2) python2-packaging-15.3-2  python2-setuptools-1:18.5-1

Total Installed Size:  1.45 MiB

:: Proceed with installation? [Y/n] 
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
checking available disk space...
installing python2-packaging...
installing python2-setuptools...
==> Retrieving sources...
  -> Downloading IMDbPY-5.0.tar.gz...
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
  -> Extracting IMDbPY-5.0.tar.gz with bsdtar
==> Starting prepare()...
==> Sources are ready.
Copied source directory to /sources/imdbpy
===> sudo -u tuscan PATH=/tmp/toolchain/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin CC=arm-linux-androideabi-gcc CXX=arm-linux-androideabi-g++ SYSROOT=/build/android-ndk-r10e/platforms/android-21/arch-arm makepkg --noextract --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: imdbpy 5.0-1 (Fri Nov 13 23:53:43 UTC 2015)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Entering fakeroot environment...
==> Starting package()...

---------------------------------------------------------------------------
This script requires setuptools version 0.6c11 to run (even to display
help).  I will attempt to download it for you (from
http://pypi.python.org/packages/2.7/s/setuptools/), but
you may need to enable firewall access for this script first.
I will start the download in 15 seconds.

(Note: if this machine does not have network access, please obtain the file

   http://pypi.python.org/packages/2.7/s/setuptools/setuptools-0.6c11-py2.7.egg

and place it in this directory before rerunning this script.)
---------------------------------------------------------------------------
Downloading http://pypi.python.org/packages/2.7/s/setuptools/setuptools-0.6c11-py2.7.egg
Traceback (most recent call last):
  File "setup.py", line 7, in <module>
    ez_setup.use_setuptools()
  File "/tmp/imdbpy/src/IMDbPY-5.0/ez_setup.py", line 103, in use_setuptools
    return do_download()       
  File "/tmp/imdbpy/src/IMDbPY-5.0/ez_setup.py", line 97, in do_download
    egg = download_setuptools(version, download_base, to_dir, download_delay)
  File "/tmp/imdbpy/src/IMDbPY-5.0/ez_setup.py", line 158, in download_setuptools
    src = urllib2.urlopen(url)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 442, in error
    result = self._call_chain(*args)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 629, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 427, in _open
    'unknown_open', req)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/tmp/toolchain/lib/python2.7/urllib2.py", line 1247, in unknown_open
    raise URLError('unknown url type: %s' % type)
urllib2.URLError: <urlopen error unknown url type: https>
==> ERROR: A failure occurred in package().
    Aborting...
===> Printing config.logs
===> Finished printing config.logs></pre></code>
        </body></html>
